CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
 # Changelog
 
+## [1.2](https://github.com/arborx/arborx/tree/v1.2) (2022-04-01)
+
+[Full Changelog](https://github.com/arborx/arborx/compare/v1.1...v1.2)
+
+**New features:**
+
+- Implement Euclidean minimum spanning tree algorithm [\#624](https://github.com/arborx/ArborX/pull/624), [\#631](https://github.com/arborx/ArborX/pull/631), [\#661](https://github.com/arborx/ArborX/pull/631)
+
+**Build changes:**
+
+- Require Kokkos 3.4 [\#578](https://github.com/arborx/ArborX/pull/578)
+
+**Enhancements:**
+
+- Indicate the status of the worktree in the hash string ("-dirty") [\#558](https://github.com/arborx/ArborX/pull/558)
+- Check valid callback and space accessibility [\#563](https://github.com/arborx/ArborX/pull/563)
+- Add a molecular dynamics example [\#564](https://github.com/arborx/ArborX/pull/564)
+- Add nightly testing against Kokkos develop [\#569](https://github.com/arborx/ArborX/pull/569)
+- Improve performance of the HIP backend when using HIP-Clang [\#575](https://github.com/arborx/ArborX/pull/575)
+- Remove PGI CI [\#590](https://github.com/arborx/ArborX/pull/590)
```

Comment:
Do you really want to mention this?  Also why would you list it under enhancement?

Reply:
I wouldn't argue that it is an enhancement with respect to maintainability to drop it. :wink: Seriously, I would either list it under "incompatibilities" or drop it.

Reply:
Dropped.